src/promptflow/promptflow/_sdk/operations/_run_submitter.py

```diff
@@ -321,12 +321,14 @@ def _submit_bulk_run(self, flow: Flow, run: Run, local_storage: LocalStorageOper
         # create run to db when fully prepared to run in executor, otherwise won't create it
         run._dump()  # pylint: disable=protected-access
         try:
-            bulk_result = batch_engine.run(
-                input_dirs=input_dirs,
-                inputs_mapping=column_mapping,
-                output_dir=local_storage.outputs_folder,
-                run_id=run_id,
-            )
+            # run executor in flow's directory because the flow default inputs path is relative to flow's directory
+            with _change_working_dir(flow.code):
```

Comment:
where did we resolve the inputs path? maybe we can pass the code dir to the place it resolves? so we don't need to do such change_working_dir.

Reply:
The resolving logic happens inside executor, and the error happens when inputs default value has path in it. Maybe we can discuss with executor if they can change the default value in flow to absolute path before execution?

Reply:
Reverted since executor fixed the issue